truffle/src/main/java/org/jruby/truffle/nodes/ext/BigDecimalNodes.java

```diff
@@ -122,6 +133,28 @@ public static RubyBasicObject createRubyBigDecimal(RubyClass rubyClass, BigDecim
         return new RubyBasicObject(rubyClass, BIG_DECIMAL_FACTORY.newInstance(Type.NORMAL, value));
     }
 
+    public static RoundingMode toRoundingMode(int constValue) {
+        switch (constValue) {
+            case 1:
+                return RoundingMode.UP;
+            case 2:
+                return RoundingMode.DOWN;
+            case 3:
+                return RoundingMode.HALF_UP;
+            case 4:
+                return RoundingMode.HALF_DOWN;
+            case 5:
+                return RoundingMode.CEILING;
+            case 6:
+                return RoundingMode.FLOOR;
+            case 7:
+                return RoundingMode.HALF_EVEN;
+        }
+
+        CompilerAsserts.neverPartOfCompilation();
+        throw new UnsupportedOperationException();
```

Comment:
I would move this in the switch `default:` case.
In general I'm in favor of indentation-based control flow (and avoid `return` in the middle) :smile: